tf_agents/environments/atari_preprocessing.py

```diff
@@ -187,8 +187,12 @@ def step(self, action):
       # We max-pool over the last two frames, in grayscale.
       elif time_step >= self.frame_skip - 2:
         t = time_step - (self.frame_skip - 2)
```

Comment:
Instead of special-casing `frame_skip == 1` below, let's redefine `t` as `t = time_step - max(self.frame_skip - 2, 0)` and add a comment that this ensures proper behavior for `frame_skip == 1`.

Reply:
Good suggestion! Sorry that somehow this fork is gone on my side. Is there any other way that I can update this PR?

Reply:
@samfishman I've created a new PR #302 . Please close this one, thanks!